be/src/Project/Services/Identity/Identity.Api/Areas/Accounts/Resources/AccountResources.pl.resx

```diff
@@ -207,6 +216,9 @@
   <data name="SignInText" xml:space="preserve">
     <value>Zaloguj się</value>
   </data>
+  <data name="SmsMarketingApprovalLabel" xml:space="preserve">
+    <value>Wyrażam zgodę na kontakt telefoniczny ze strony ELTAP w celach marketingowych</value>
```

Comment:
**suggestion:** Word ELTAP couldn't be in text, because this app is not used only by ELTAP